flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/proxy/KinesisProxy.java

```diff
@@ -253,6 +262,35 @@ public String getShardIterator(KinesisStreamShard shard, String shardIteratorTyp
 		return getShardIteratorResult.getShardIterator();
 	}
 
+	/**
+	 * Determines whether the exception can be recovered from using
+	 * exponential-backoff
+	 * 
+	 * @param ex
+	 *            Exception to inspect
```

Comment:
nit: I think the Javadoc formatting here is inconsistent with the other methods (line change).